doc/tutorials/3d/point_cloud/point_cloud.markdown

```diff
@@ -0,0 +1,75 @@
+Point cloud visualisation {#tutorial_point_cloud}
+==============================
+
+|    |    |
+| -: | :- |
+| Original author | Klepikov Dmitrii |
+| Compatibility | OpenCV >= 5.0 |
+
+Goal
+----
+
+In this tutorial you will:
+
+-   Load and save point cloud data
+-   Visualise your data
+
+Requirements
+------------
+
+For visualisations u need to complile OpenCV library with OpenGL support. For this you should set WITH_OPENGL flag on in cmake
+
+Practice
+-------
+
+Loading and saving of point cloud can be done using loadPointCloud(savePointCloud)
+
+Currently supported formats are:
+- .OBJ (supported keys are v(which is responcible for point position), vn(normal coordinates) and f(faces of a mesh), other keys are ignored)
+- .PLY (all encoding types(ascii and byte) are supported with limitaion to only float type for data)
+
+@code{.py}
+vertices, normals = cv2.loadPointCloud("teapot.obj")
```

Comment:
I recommend to use code substitution from example python file as it's done in `tutorials/imgproc/histograms/back_projection/back_projection.markdown` and `python/tutorial_code/Histograms_Matching/back_projection/calcBackProject_Demo1.py`. The samples code is checked with CI and is always "buildable" with the library.